### PR TITLE
Scanning of Pokestop Name, Image, Desc

### PR DESCRIPTION
**Pull request by KoffeinKaio**
## Description
Implemented scanning of pokestop details just like -gi does, every time a scanner parses a pokestop, it looks in the db of information is missing and gets them.
With the -pi switch you can enable Pokestop details scanning, with the -pie you can set the expire time (the time it gets rescanned) in minutes. (Does a pokestop even change?)

## Motivation and Context
I have a really bad orientation, so I wanted to see the Images of Pokestops.
Also people tend to say "hey, Pokestop is lured!" not "Hey Pokestop at [lat, lng] is Lured!"

## How Has This Been Tested?
tested all features on my ded. server

## Screenshots (if appropriate):
pokestop

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by CoolCanuck** [comment]
@KoffeinKaio Does lure countdown timer work? I couldn't get it to work :(

**Comment by KoffeinKaio** [comment]
On the old PR or with this one?

**Comment by CoolCanuck** [comment]
Oops. Old one. Sorry

File: pogom/models.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 flaskDb = FlaskDB()
 cache = TTLCache(maxsize=100, ttl=60 * 5)
 
-db_schema_version = 7
+db_schema_version = 8
```
Comment (jnovack) [inline pogom/models.py]: Rebase.

File: static/js/map.js [hunk 2]
```diff
@@ -425,35 +425,47 @@ function gymLabel (teamName, teamId, gymPoints, latitude, longitude, lastScanned
...
+  }
+
+  if (item.expireTime) {
+    var expireDate = new Date(item.expireTime)
 
     str = `
-      <div>
+      <div style="text-align: center;">
         <b>Lured Pokéstop</b>
+        ${pkstpinfo}
```
Comment (jnovack) [inline static/js/map.js]: The name is the most important thing, that should be on TOP and **BOLD**, the type should be _italicized_

**Niantic Headquarters**
_Lured Pokéstop_

NOT

**Lured Pokéstop**
**Niantic Headquarters**

Comment (titanicboyke) [inline static/js/map.js]: I changed the files and I get an error!
`Traceback (most recent call last):
  File "C:\Users\user\Desktop\PokemonGo\runserver.py", line 291, in <module>
    main()
  File "C:\Users\user\Desktop\PokemonGo\runserver.py", line 222, in main
    if not args.disable_clean:
AttributeError: 'Namespace' object has no attribute 'disable_clean'`

File: pogom/models.py (hunk 1 above)
Comment (KoffeinKaio) [inline pogom/models.py]: 7 is still the current db_version in the develop branch, why rebase?

**Comment by venimo** [comment]
The pokestop is lured but there isn't information about the expiration.

**Comment by venimo** [comment]
Can you take a look @KoffeinKaio

**Comment by KoffeinKaio** [comment]
No time for the next 2 days, will take a look after.

File: pogom/models.py (hunk 1 above)
Comment (jnovack) [inline pogom/models.py]: Because this.

image

**Comment by primaxius** [comment]
conflicts and declined.

**Pull request closed without merging**